src/Configuration.ts

```diff
@@ -82,6 +82,7 @@ export interface ExchangeConfiguration {
   readonly exchangeStorageWriterStoreNextClaim: string
   readonly exchangeGetHealth: string
   readonly exchangePurgeStaleTransactions: string
+  readonly exchangeBlockchainReaderForkDetected: string
```

Comment:
Let's keep module names out of the exchange names. `exchangeForkDetected` is fine.